pkg/chunk/cache/memcached_client.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // MemcachedClient interface exists for mocking memcacheClient.
 type MemcachedClient interface {
-	GetMulti(keys []string) (map[string]*memcache.Item, error)
+	GetMulti(keys []string, opts ...memcache.Option) (map[string]*memcache.Item, error)
```

Comment:
This interface is used in results cache only IIUC. Do we need to make any changes on Thanos memcached client?

Reply:
yeah.. i just realized that...